tox.ini

```diff
@@ -7,13 +7,41 @@ deps = nose
        PyYAML
        six
        packaging
+       cython
+       eventlet
+       twisted <15.5.0
 
 [testenv]
 deps = {[base]deps}
-       cython
-       py{27}: gevent
-       twisted <15.5.0
+
 setenv = LIBEV_EMBED=0
          CARES_EMBED=0
+         MONKEY_PATCH_LOOP=0
 changedir = {envtmpdir}
 commands = nosetests --verbosity=2 --no-path-adjustment {toxinidir}/tests/unit/
+
+
+[testenv:py27]
+deps = {[base]deps}
+       gevent
+
+setenv = LIBEV_EMBED=0
+         CARES_EMBED=0
+         MONKEY_PATCH_LOOP=1
+changedir = {envtmpdir}
+commands =
```

Comment:
It'd be nice to find a way around this code duplication, the docs from tox seem to do something similar though: http://tox.readthedocs.io/en/latest/example/unittest.html